var/spack/repos/builtin/packages/r-rhtslib/package.py

```diff
@@ -39,3 +39,4 @@ class RRhtslib(RPackage):
 
     depends_on('r@3.4.0:3.4.9', when='@1.8.0')
     depends_on('r-zlibbioc', type=('build', 'run'))
+    depends_on('autoconf@2.69', type='build')
```

Comment:
> version 2.67 or higher

so it should be `depends_on('autoconf@2.69:', type='build')`

Reply:
Hi @davydden, thanks for spotting this! I just updated the PR (included a new commit).